README.md

```diff
@@ -89,7 +89,9 @@ There is also a demo application:
 
 ## Build on Mac OS X
 
-The following instructions were verified with Mac OS X El Capitan.
+The following instructions were verified with Mac OS X El Capitan. Alternatively
```

Comment:
To clarify my previous comment: let's remove all the manual steps listed in this section, and instead only suggest running the `install_osx_dependencies.sh` script.  (Otherwise, I think, we're begging for one or the other to get out of sync in the long run.)

We can still give a summary of what the script does in this section; for example:

    The script installs dependencies via [Homebrew](http://brew.sh) and then
    downloads and installs the TensorFlow library on your machine under
     `/usr/local`.